src/apps/rDSN.monitor/app_package/MonitorApp.py

```diff
@@ -449,11 +449,6 @@ def post(self):
             savedFile.write(raw_file)
             savedFile.close()
             
-            #TODO: shall we need some mechanism to detect if 7z are truely installed?
```

Comment:
The unzip process is no longer at the monitor but is migrated to deployment server. So maybe this line should be deleted from here and we could implement such logic at deployment server. For example, maybe the client command of deployment should return a description of the failure cause?